packages/unpublished-test-infra/addon-test-support/watch-property.js

```diff
@@ -1,6 +1,13 @@
 import { removeObserver, addObserver } from '@ember/object/observers';
 import QUnit from 'qunit';
 
+function freeze(obj) {
```

Comment:
Let's import this function from one freeze.ts file

Reply:
We may want to do this long term but we would have a separate util for runtime vs tests. currently we only use freeze in one location in tests (here) and two locations in store. The locations in store could possible share a definition but both are DEBUG only and thus stripped away, likely we'll refactor it to a util if we hit a third case.